File: atcoder/abc176A_takoyaki.ml
(*
 * Vicfred
 * https://atcoder.jp/contests/abc176/tasks/abc176_a
 * implementation
 * *)
Scanf.scanf "%d %d %d" (fun n x t ->
    Printf.printf "%d\n" (((n + x - 1)/ x) * t))

File: atcoder/abc176A_takoyaki.nim
# uninhm
# https://atcoder.jp/contests/abc176/tasks/abc176_a
# math

import strutils, sequtils

var n, x ,t: int
(n, x, t) = stdin.readLine.split.map(parseInt)

echo (n + x - 1) div x * t
